magento/pwa-studio
# Ticket #3209. Adds check by include_in_menu attribute before appendin…

…g children categories to the returned array.

## Description

Adds a check to avoid returning children categories when doesn't apply.


Closes #3209 

## Breaking Changes

Breaking because it now does not display content if `include_in_menu` is false.

### Verification Steps

1. Go to the Categories edit form in the admin and make some of them **include_in_menu** true, and some false.
2. Go to your pwa and open the mobile devices view.
3. Make sure you only see the categories that have **include_in_menu** true.
